lib/web_ui/lib/src/engine/configuration.dart

```diff
@@ -312,6 +312,19 @@ class FlutterConfiguration {
   /// The font used to render color emojis is large (~24MB). This configuration
   /// gives developers the ability to decide for their app.
   bool get useColorEmoji => _configuration?.useColorEmoji ?? false;
+
+  /// Whether to load fonts from the font manifest prior to calling the app's
+  /// `main` function.
+  ///
+  /// If set to true, manifest fonts will load as part of engine initialization,
+  /// delaying the invocation of `main`, if necessary.
+  ///
+  /// If set to false, engine initialization does not include manifest fonts.
+  /// Instead, the developer can load manifest fonts using the
+  /// `loadManifestFonts` function from `dart:ui_web`.
+  ///
+  /// Defaults to true.
+  bool get loadManifestFontsBeforeAppMain => _configuration?.loadManifestFontsBeforeAppMain ?? true;
```

Comment:
Is the concept of the "manifest" exposed to app developers? Looking at the external documentation e.g. https://docs.flutter.dev/cookbook/design/fonts, it seems like developers don't usually touch it, and 1P developers ideally shouldn't either.

How about just calling this 'loadFontsBeforeAppMain' and the other function 'loadFonts' ? Or is that too generic?

Reply:
I wanted to qualify this with _something_ because it does not affect all fonts. For example, fallback fonts will still attempt to load if you try rendering a piece of text. You can also load a font using the `FontLoader`. This change only affects the fonts declared in [pubspec](https://docs.flutter.dev/cookbook/design/fonts#2-declare-the-font-in-the-pubspec). There's no special term used in pubspec for those fonts. I've heard them called "asset fonts" and "custom fonts", none of which imply "fonts that are loaded prior to the app". As of today calling them "pubspec fonts" may actually be accurate, but I'm not sure if it would be future-proof. We could add fonts to the manifest from other sources in the future. For example, we discussed generating a minimal fallback font bundle based on predeclared strings that appear in the app.

Reply:
I see this isn't as simple as I thought! I like Mouad's "loadManifestFontsManually" suggestion.

Reply:
I like `loadManifestFontsManually`! I'm willing to remove the reference to the "manifest" if that feels off, but I still like that the word "manifest" (aside from its specific usages in Flutter today) implies that you have declared a list of fonts somewhere and it's a list that's known to the Flutter runtime and has special meaning that's different from other fonts that exist in your app. LMK.

Reply:
I'm fine with 'manifest' now I understand more context.